src/ofxstatement/tool.py

```diff
@@ -149,7 +149,21 @@ def convert(args):
         statement = parser.parse()
     except exceptions.ParseError as e:
         log.error("Parse error on line %s: %s" % (e.lineno, e.message))
-        return 2  # error
+        return 2  # parse error
+    except exceptions.ValidationError as e:  # any assert_valid() may raise it
+        appui.warning("Problem with user input")
+        # log the exception with a traceback
+        log.exception(e)
+        return 3  # validation error
+
+    # Validate the statement
+    try:
+        statement.assert_valid()
+    except exceptions.ValidationError as e:
+        appui.warning("Problem with user input")
+        # log the exception with a traceback
+        log.exception(e)
```

Comment:
Again, let's not show tracebacks to the user, if it is not a bug in ofxstatement.

Reply:
What do you suggest? I just want the message plus the contents of the object which has an error. Do you want the handling removed?